src/flowchart/setResponseMessage.md

```diff
@@ -0,0 +1,27 @@
+# Set response message
+Sets the response message for the command.
+
+## Inputs
+- `Content` `(Type: String || Own text: true)`: Content of the sent message.
+
+## Flags
+- ➤ Execute
+- 🗑 Delete
```

Comment:
Delete is available on all functions except entry point, no point in mentioning that